File: Practice problem/continuous duplicate deletion/continuous duplicate deletion.py
# 연습문제 같은 숫자는 싫어 Level 1

def solution(arr):
    answer = []
    answer.append(arr[0])
    for i in range(0, len(arr)):
        if answer[len(answer) - 1] != arr[i] :
            answer.append(arr[i])
    return answer

File: Practice problem/continuous duplicate deletion/continuous duplicate deletion.cpp
// 연습문제 같은 숫자는 싫어 Level 1

#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> arr)
{
	vector<int> answer;
	answer.push_back(arr[0]);
	for (int i = 1; i < arr.size(); i++) {
		if (answer[answer.size() - 1] != arr[i]) { answer.push_back(arr[i]); }
	}
	return answer;
}
